multiqc/modules/jellyfish/jellyfish.py

```diff
@@ -0,0 +1,96 @@
+#!/usr/bin/env python
+
+""" MultiQC module to parse results from jellyfish  """
+
+from __future__ import print_function
+
+from collections import OrderedDict
+import logging
+from multiqc import config
+from multiqc.plots import linegraph, bargraph
+from multiqc.modules.base_module import BaseMultiqcModule
+
+
+
+# Initialise the logger
+log = logging.getLogger(__name__)
+
+
+class MultiqcModule(BaseMultiqcModule):
+    def __init__(self):
+        # Initialise the parent object
+        super(MultiqcModule, self).__init__(name='jellyfish', anchor='jellyfish',
+        href="http://www.cbcb.umd.edu/software/jellyfish/",
+        info="JELLYFISH is a tool for fast, memory-efficient counting of k-mers in DNA.")
```

Comment:
For historical reasons, the first word of `info` is the `name` from above. So this reads _"jellyfish JELLYFISH is a.."_. So need to make it `info="is a tool for fast.."`.

Also maybe make the `name` sentence case? `name='Jellyfish'`